docs/logging.md

```diff
@@ -235,20 +221,28 @@ to resolve a program. When the interpreter starts, the RESL channel logs the lis
 of strategies used to locate a RPG/Java program.
 
 ```
-15:09:46.910                  RESL resource: /
-15:09:46.910                  RESL directory: .
-15:09:46.960 TEST_06       80 RESL CALL "CALCFIB"
-+-----------+-------------+--+---+---------- resolution -----------+
+15:47:40.824 RESL	T10_A60_P02	8	CALL	"MULANGTC10"
++------------ header -------------+---- resolution ----+
```

Comment:
Possibly due to a copy/paste, I don't see the information about what are:
- 451457us
- 33%
- 1
- avg. 451457us